main.rb

```diff
@@ -0,0 +1,68 @@
+require_relative 'handlers'
+require './book_ui'
+require './person_ui'
+require './rental_ui'
+
+class App
```

Comment:
Please be sure to create an app.rb file that will serve as their console app entry-point 🤓 You can check this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/console_apps.md#console-app-entry-point) if you need guidance.

Reply:
@mateo951  check on Handlers is included at  App method.